qds_sdk/sensors.py

```diff
@@ -0,0 +1,75 @@
+"""
+The sensors module contains the base definition for a generic
+sensor call and the implementation of all the specific sensors
+"""
+
+from __future__ import print_function
+from qds_sdk.qubole import Qubole
+from qds_sdk.resource import Resource
+from argparse import ArgumentParser
+
+import logging
+import json
+
+log = logging.getLogger("qds_sensors")
+
+
+class SensorCmdLine:
+
+    @staticmethod
+    def check(cls, args):
```

Comment:
NIT: Need to rename the variable from cls to sensor_class. The \@staticmethod along with cls as variable name is confusing.